Add Perez2022

Darcy220606 opened the pull request:
# Pull Request

This PR is for a

- [x] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [x] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental), ask **@thedir-team-dirt** for advice)
- [ ] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated), ask **@thedir-team-bugparty** for advice)
- [ ] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated), ask **@thedir-team-pathopeeps** for advice)

## New Publication

### Publication Information

This pull request is to add samples from the following publication(s): <!-- REPLACE this comment (including the < > symbols) with citation(s) URLs/DOIs/descriptions-->

This is to close <!-- REPLACE this comment (including the < > symbols) with a hashtag and the corresponding issue number here, e.g. #10 -->

### Checklist

- [x] Publication is published (preprints currently not accepted)?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_ (newly re-sequenced metagenomes are OK!)?
- [x] Samples are shotgun metagenomes (hostassociated-singlegenome may also contain whole-genome enriched data)?
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [x] Changelog is updated to include the publication under 'Added'?
- [ ] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [x] Review requested from corresponding team (see list of lists above for each team)?
- [x] (If applicable) Made a separate PR to add new categories to controlled lists (stored under `/assets/enums`, e.g. material or archive)? If so the corresponding PR is here: <!-- REPLACE this comment  (including the < > symbols) with that PR -->

Darcy220606 commented:
The authors mention two extreme sites: `hydrothermal system` and `wetlands` and no other information besides they are terrestrial samples. So i added those two features in the enum `features` as the other options are too specific.

jfy133 commented:
> To @SPAAM-community/thedir-team-dirt. The authors mention two extreme sites: `hydrothermal system` and `wetlands` and no other information besides they are terrestrial samples. So i added those two features in the enum `features` as the other current options are too specific.

Ok, sounds good to me - however you need to do this as a separate PR into master!

(sorry for the delay getting back to you on this @Darcy220606!)

jfy133 commented on ancientmetagenome-environmental/samples/ancientmetagenome-environmental_samples.tsv:
```diff
@@ -467,3 +467,23 @@ Iwanska2022	2022	10.1038/s41598-022-12761-w	Lake Slone	51.305	23.366	Poland	flor
+Perez2022	2022	10.3390/microorganisms10081623	Lirima hydrothermal system	-19.854	-68.908	Chile	microbial	S3	110	3PHCH3	1100	10.3390/microorganisms10081623	hydrothermal system	soil	NA	SRA	PRJNA794308	SRS11699657
+Perez2022	2022	10.3390/microorganisms10081623	Lirima hydrothermal system	-19.854	-68.908	Chile	microbial	S3	110	3PB2	1100	10.3390/microorganisms10081623	hydrothermal system	soil	NA	SRA	PRJNA794308	SRS11699655
+Perez2022	2022	10.3390/microorganisms10081623	Lirima hydrothermal system	-19.854	-68.908	Chile	microbial	S3	110	3CP2	1100	10.3390/microorganisms10081623	hydrothermal system	soil	NA	SRA	PRJNA794308	SRS11699653
+Perez2022	2022	10.3390/microorganisms10081623	 Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4QG2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699664
+Perez2022	2022	10.3390/microorganisms10081623	 Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PL2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699662
+Perez2022	2022	10.3390/microorganisms10081623	 Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PHCH3	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699660
+Perez2022	2022	10.3390/microorganisms10081623	 Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PB2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699661
+Perez2022	2022	10.3390/microorganisms10081623	 Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4CP2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699659
```
```suggestion
Perez2022	2022	10.3390/microorganisms10081623	Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4QG2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699664
Perez2022	2022	10.3390/microorganisms10081623	Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PL2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699662
Perez2022	2022	10.3390/microorganisms10081623	Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PHCH3	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699660
Perez2022	2022	10.3390/microorganisms10081623	Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4PB2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699661
Perez2022	2022	10.3390/microorganisms10081623	Salar de Huasco wetlands	-20.287	-68.891	Chile	microbial	S4	80	4CP2	800	10.3390/microorganisms10081623	wetland	soil	NA	SRA	PRJNA794308	SRS11699659
```

jfy133 reviewed (changes requested):
Mostly improving precision in the samples table :)

# General

- Missing CHANGELOG update (we are doing these now again ;) )

# Samples

- What you have as `sequence` now should be `sample_name` (the current `sample_name` are actually extract IDs)! Therefore you need to collapse all the rows into just the four samples S1-S4, and provide a comma separated list in the `archive_accession`. Note you _don't_ need to collapse libraries!
- You have a space before `Salar` (see suggestion)
- Lat:Lon, when you copy/paste the provided Lat:Lons into google it gives you an exact one underneath:
    - Lirima - S1: -19.851, -68.906; S2:  -19.851, -68.906 (etc.)
    - Husaco - the Longitude shoud be -68.888 
 - Study focus should be `faunal,floral,microbial` as they actually also screen for eukarotes 
- Sequence should be 'unknown' (although should probably add something called 'none' instead..., but I 'll do that in the future)
- For features, we need to match with ENVO:
    - For Lirima I suggest: [Hydrothermal field](https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_01001854)
    - For Salar de Huasco  I suggest [Wetland area](https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_00000043)

# Libraries

- `sample_name` should match the S1-S4, but `library_name` is fine now
- `library_treatment` should be 'none' (as if they don't mention anything explicitly, it'll be none

Darcy220606 commented:
> > To @SPAAM-community/thedir-team-dirt. The authors mention two extreme sites: `hydrothermal system` and `wetlands` and no other information besides they are terrestrial samples. So i added those two features in the enum `features` as the other current options are too specific.
> 
> Ok, sounds good to me - however you need to do this as a separate PR into master!
> 
> (sorry for the delay getting back to you on this @Darcy220606!)

Thanks @jfy133 . I made a new PR #1033 to add it and deleted it here
